src/pages/java/arrays/index.md

```diff
@@ -51,6 +51,33 @@ dataType[] name_of_array = {value0, value1, ..., valuek};
 double[] list = {1, 2, 3, 4};
 ```
 
+## Accessing Arrays:
+```java
+arrayName[index]; //gives you the value at the specified index
```

Comment:
Can you add a space between the double slashes and the comment ?   
 ```java
// gives you the value...
```
This rule is contained in the article style guide in the README of this project: https://github.com/freeCodeCamp/guides#formatting-example-code